keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -522,8 +531,12 @@ in back-to-back releases.
 #### Alpha
 
  * Feature functionality implemented but hidden behind a feature gate
- * Add unit, functional, upgrade and downgrade tests to automated k8s test.
+ * Add unit, e2e and functional tests to automated k8s test.
```

Comment:
unit and integration for alpha, e2e for beta?

Reply:
Done